packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -391,11 +398,14 @@ export async function getBookings({
       // We need to get the sorted bookings here as well because plainBookings array is not correctly sorted
       orderBy,
     })
-  ).map((booking) => {
+  ).map(async (booking) => {
     // If seats are enabled and the event is not set to show attendees, filter out attendees that are not the current user
     if (booking.seatsReferences.length && !booking.eventType?.seatsShowAttendees) {
       booking.attendees = booking.attendees.filter((attendee) => attendee.email === user.email);
     }
+    const bookerBaseUrl = booking.eventType?.team?.parentId
+      ? await getBookerBaseUrl(booking.eventType?.team?.parentId)
+      : await getBookerBaseUrl(booking.user?.movedToProfile?.organizationId ?? null);
```

Comment:
Let's abstract it out something like this
```suggestion
    const bookerBaseUrl = getBookerBaseUrl(getOrganizationIdOfBooking() ?? null);
```

Reply:
Done